Lib/functools.py

```diff
@@ -236,14 +236,14 @@ def __ge__(self, other):
 
 def reduce(function, sequence, initial=_initial_missing):
     """
-    reduce(function, sequence[, initial]) -> value
+    reduce(function, iterable[, initial]) -> value
 
-    Apply a function of two arguments cumulatively to the items of a sequence,
-    from left to right, so as to reduce the sequence to a single value.
-    For example, reduce(lambda x, y: x+y, [1, 2, 3, 4, 5]) calculates
+    Apply a function of two arguments cumulatively to the items of a sequence
+    or iterable, from left to right, so as to reduce the iterable to a single
+    value.  For example, reduce(lambda x, y: x+y, [1, 2, 3, 4, 5]) calculates
```

Comment:
```suggestion
    value. For example, reduce(lambda x, y: x+y, [1, 2, 3, 4, 5]) calculates
```

Reply:
@ZackerySpytz  nit :)

Reply:
There are other comments/docstrings in functools that have 2 spaces after a period, so I think it's OK as is.

Reply:
@csabella got it